js/react_native/lib/binding.ts

```diff
@@ -64,6 +64,7 @@ export declare namespace Binding {
 
   interface InferenceSession {
     loadModel(modelPath: string, options: SessionOptions): Promise<ModelLoadInfoType>;
+    loadModelFromBase64EncodedBuffer?(buffer: string, options: SessionOptions): Promise<ModelLoadInfoType>;
```

Comment:
Do we have any react native tests that can be updated to call this?

Reply:
As discussed. Changes were manually verified in an expo app. Need to update our e2e RN tests in the next week or two but want this change to make the release.